mLRS/CommonRx/out_interface.h

```diff
@@ -33,7 +37,9 @@ class tOut : public OutBase
     {
         OutBase::Init(&Setup.Rx);
         out_init_gpio();
+#ifndef ESP32
```

Comment:
this is not so nice
three possible suggestions
- add this function to the uart lib and let it just do the normal init (i.e. add to uart_template.h)
- add an empty prototype in the top of this file where there is the branch for esp32 anyway
- add an empty prototype to the hal file
I guess I like (1) best